src/install.rst

```diff
@@ -227,16 +227,15 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
       .. hint::
 
-         By default, your Windows home directory will not be directly accessible under your WSL home directory. When run in a WSL prompt, the following command fixes that by creating a symlink to it in your WSL home directory. This allows you to use Windows-based text editors and Linux commands all on the same files. Before running,
-
-         1. Replace ``<user>`` with your Windows username.
-         2. Optionally, change the ``windows_home`` folder name or only link to a specific directory under your windows user (e.g. ``ln -s /mnt/c/Users/<user>/Documents ~/windows_documents``).
+         By default, your Windows home directory will not be directly accessible under your WSL home directory. When run in a WSL prompt, the following command fixes that by creating a symlink to it in your WSL home directory. This allows you to use Windows-based text editors and Linux commands all on the same files.
 
          .. code-block:: bash
 
-               ln -ws /mnt/c/Users/<user> ~/windows_home
+               ln -ws $(wslpath $(wslvar USERPROFILE)) ~/windows_home
```

Comment:
This needs quotes to be safe. 

```suggestion
               ln -ws "$(wslpath "$(wslvar USERPROFILE)")" ~/windows_home
```

Reply:
Done: c7d27becfddafa7f4ddb1fb75d86af715bb1de8b

Reply:
Forgot to apply this to the alternative example. Done: 9e88c29d60c80006efc3da643e3e762a5aceed1a